app/decorators/user_decorator.rb

```diff
@@ -77,6 +77,14 @@ def password_reset_profile
     profile if profile&.password_reset?
   end
 
+  def threatmetrix_review_pending?
+    @threatmetrix_review_pending ||= threatmetrix_review_pending_profile.present?
+  end
+
+  def threatmetrix_review_pending_profile
+    user.profiles.threatmetrix_review_pending.order(created_at: :desc).first
```

Comment:
What's the point of ordering these?

Reply:
Just to get the most recent one. There _should_ only be one but want to make sure we are consistent if somehow there are multiple.